src/app/publicSettings.js

```diff
@@ -17,4 +17,5 @@ export default {
     appId: process.env.APP_ID || '1:966661181602:web:0b0911b19a3cefe21bc7fb',
     measurementId: process.env.MEASUREMENT_ID || 'G-QFC4GSCXKR',
   },
+  GOOGLE_API_KEY: 'AIzaSyCorQm3g4mBVQQdaoVKxB4kYto3v5WIiD0',
```

Comment:
Why not add the same pattern as the others where you try to get it from the environment first?

Reply:
I don't think this is a major concern, as we won't have different values here for staging and production.